actors/install-actor.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, type Page } from '@playwright/test';
+import { MainPage } from '../pages/main-page';
+import { ConfirmInstallationPage } from '../pages/confirm-installation-page';
+import { InstallationProgressPage } from '../pages/installing-page';
+import { CongratulationsPage } from '../pages/installation-finsihed-page';
+
+export class InstallActor {
+    readonly page: Page;
+    readonly mainPage: MainPage;
+
+    constructor(page: Page, mainPage: MainPage) {
+        this.page = page;
+        this.mainPage = mainPage;
+    }
+
+    async handleInstallation() {
+        // start the installation
+        await this.mainPage.install();
+        const confirmInstallationPage = new ConfirmInstallationPage(this.page);
+        await confirmInstallationPage.confirm();
+        // wait for the package installation progress
```

Comment:
another obvious comment, please drop it, it is not bad per se, but it creates an example that end up degenerating, with good naming in methods we are really fine, only comments for exceptions.